recipes/mdspan/all/conanfile.py

```diff
@@ -1,8 +1,11 @@
-from conans import ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile, tools
```

Comment:
```suggestion
from conan import ConanFile
```
It's not allowed to use conan.tools directly, but it's just leftover here, as I can see.